packages/client/src/components/shop/DetailSheet.tsx

```
@@ -0,0 +1,299 @@
+    <div
+      onClick={onClose}
+      style={{
+        position: 'fixed',
+        inset: 0,
+        zIndex: 50,
+        background: 'rgba(4,21,14,0.85)',
+        display: 'flex',
+        alignItems: 'flex-end',
+        justifyContent: 'center',
+      }}
+    >
```

Biome (2.5.0)

[error] Unexpected event handler on static element.

(lint/a11y/noStaticElementInteractions)